pinot-common/src/main/java/org/apache/pinot/common/datablock/DataBlockUtils.java

```diff
@@ -62,13 +67,13 @@ public static MetadataBlock getEndOfStreamDataBlock() {
     return new MetadataBlock(MetadataBlock.MetadataBlockType.EOS);
   }
 
-  public static MetadataBlock getEndOfStreamDataBlock(Map<String, String> stats) {
-    return new MetadataBlock(MetadataBlock.MetadataBlockType.EOS, stats);
+  public static int readVersionType(ByteBuffer byteBuffer) {
+    return byteBuffer.getInt();
```

Comment:
This method is here to be called from a test, but can be useful in other cases in the future.